modules/@apostrophecms/image/ui/apos/components/AposMediaManagerEditor.vue

```diff
@@ -76,11 +76,11 @@
         />
         <AposButton
           v-if="activeMedia._id && !restoreOnly"
-          @click="trash"
+          @click="archive"
           icon="trash-can-icon"
```

Comment:
This will be changing, right? So we don't need to change it now.

Reply:
I am not sure what you mean by "this will be changing," but this PR should be internally consistent and everything that worked before it should work after it.

Reply:
The button will be moving from this location, so we don't need to worry about switching the icon.